Author: Simon Long <simon@raspberrypi.org>
Forwarded: https://patchwork.kernel.org/project/bluetooth/patch/20201218190609.107898-4-dave.jones@canonical.com/
Last-Update: 2022-12-06
Description: Patches for loading the bcm43xx firmware
 Disables setting the UART interface speed *before* loading the firmware
 (a later call sets the speed of the interface as requested).

Index: bluez-5.71/tools/hciattach_bcm43xx.c
===================================================================
--- bluez-5.71.orig/tools/hciattach_bcm43xx.c
+++ bluez-5.71/tools/hciattach_bcm43xx.c
@@ -352,9 +352,6 @@
 	if (bcm43xx_locate_patch(HCI_TO_STR(FIRMWARE_DIR), chip_name, fw_path)) {
 		fprintf(stderr, "Patch not found, continue anyway\n");
 	} else {
-		if (bcm43xx_set_speed(fd, ti, speed))
-			return -1;
-
 		if (bcm43xx_load_firmware(fd, fw_path))
 			return -1;
 
